frontend/src/components/Forms/SignupForm.jsx

```diff
@@ -0,0 +1,206 @@
+import { Card, Divider, Grid, Typography, CardContent, Button, TextField } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import FacebookIcon from '../../images/facebook-icon.svg';
+import GoogleIcon from '../../images/google-icon.svg';
+
+import api from '../../helpers/api';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    maxWidth: '100%',
+    width: 500,
+    margin: '0 auto',
+    overflowY: 'auto',
+  },
+  header: {
+    padding: theme.spacing(3),
+    [theme.breakpoints.down('xs')]: {
+      display: 'flex',
+      flexDirection: 'column',
+      textAlign: 'center',
+    },
+  },
+  headerLogin: {
+    [theme.breakpoints.down('xs')]: {
+      marginBottom: theme.spacing(2),
+    },
+  },
+  signUpLink: {
+    color: theme.palette.primary.main,
+  },
+  content: {
+    padding: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+  },
+  or: {
+    backgroundColor: theme.palette.common.white,
+    margin: '-42px auto 42px',
+    width: '100px',
+    fontSize: '16px',
+    color: theme.palette.grey['700'],
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  divider: {
+    marginTop: theme.spacing(4),
+    marginBottom: theme.spacing(4),
+    width: '100%',
+    maxWidth: 316,
+  },
+  input: {
+    marginBottom: theme.spacing(4),
+    fontSize: '16px',
+    lineHeight: '20px',
+    maxWidth: 316,
+    width: '100%',
+  },
+  loginButton: {
+    backgroundColor: theme.palette.primary.main,
+    borderRadius: 40,
+    maxWidth: 316,
+    width: '100%',
+  },
+  externalLoginButton: {
+    marginBottom: theme.spacing(2),
+    marginTop: theme.spacing(2),
+    display: 'flex',
+    justifyContent: 'space-around',
+    border: '1px solid #2196F3',
+    borderRadius: '40px',
+    color: theme.palette.info.main,
+    letterSpacing: 1,
+    maxWidth: 316,
+    width: '100%',
+  },
+  forgotPassword: {
+    marginBottom: theme.spacing(3),
+    color: theme.palette.primary.main,
+    maxWidth: 316,
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'flex-end',
+  },
+  error: {
+    marginBottom: theme.spacing(2),
+    color: '#F44336',
+  },
+}));
+
+const SignupForm = (props) => {
+  const { closeLogin, history } = props;
+
+  const classes = useStyles();
+
+  const [email, setEmail] = React.useState('');
+  const [password, setPassword] = React.useState('');
+  const [emailError, setEmailError] = React.useState('');
+  const [passwordError, setPasswordError] = React.useState('');
+  const [loginError, setLoginError] = React.useState('');
+
+  const handleSubmit = async () => {
+    setEmailError('');
+    setPasswordError('');
+    setLoginError('');
+
+    if (email.length <= 0) {
+      setEmailError('Email is required');
+      return;
+    }
+
+    if (password.length <= 0) {
+      setPasswordError('Password is required');
+      return;
+    }
+
+    const response = await api.login({ username: email, password });
```

Comment:
If this is signing up for an account, it'll need to call to the [/users/](https://github.com/crowdsourcemedical/volunteer-database/blob/master/backend/app/routers/user.py#L44-L62) endpoint, not login.